src/Google/RequestReviewStatuses.php

```diff
@@ -35,7 +35,8 @@ public function get_statuses_from_response( array $response ) {
 					isset( $region_status['reviewIneligibilityReasonDetails'] ) &&
 					isset( $region_status['reviewIneligibilityReasonDetails']['cooldownTime'] )
```

Comment:
❓

Just looking at the response shared in the PR, should we not check if ReviewIneligibilityReason === 'IN_COOLDOWN_PERIOD' and not only if the field cooldownTime exists?

Reply:
Based on the API specifications seems like that field only appears in case of cool down period 

"This timestamp represents end of cooldown period for review ineligibility reason IN_COOLDOWN_PERIOD."

Info: https://developers.google.com/shopping-content/reference/rest/v2.1/freelistingsprogram#ReviewIneligibilityReasonDetails

Reply:
Thanks, @puntope for the clarification!